{"name": "Test đăng nhập với dữ liệu từ Excel", "status": "broken", "statusDetails": {"message": "KeyError: 'username'", "trace": "E   KeyError: 'username'\nAll traceback entries are hidden. Pass `--full-trace` to see hidden and internal frames."}, "parameters": [{"name": "login_data", "value": "('abc@gmail.com', 543215, 'fail')"}], "start": 1751536918563, "stop": 1751536918564, "uuid": "6fedd3c1-3299-47f9-912f-667d9a357344", "historyId": "cbe5512b90c68dab7699532d75aa0f6a", "testCaseId": "8795e2619fcc855f6c4045aeba8c6c68", "fullName": "tests.test_login#test_login_from_excel", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "host", "value": "SPC"}, {"name": "thread", "value": "11420-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}